certified-connectors/LMS365/apiDefinition.swagger.json

```diff
@@ -2661,6 +2758,111 @@
         ]
       }
     },
+    "/webhooks/subscribe/CourseCreated": {
+      "x-ms-notification-content": {
+        "description": "Response",
+        "schema": {
+          "type": "object",
+          "properties": {
+            "Id": {
+              "type": "string",
+              "description": "Id"
+            },
+            "Attempt": {
+              "type": "integer",
+              "format": "int32",
+              "description": "Attempt"
+            },
+            "Properties": {
+              "type": "object",
+              "properties": {},
```

Comment:
@elearningforce-ag There are many empty `"properties"` and `"schema"` fields, Can you please remove them all of them, if they do not add any value to the connector